test_util/helpers.py

```diff
@@ -31,6 +30,35 @@ def authenticate(self, cluster):
         logging.info('Authentication successful')
 
 
+class ApiClient:
+    def __init__(self, cluster, user, api_base, default_headers={}):
+        self.cluster = cluster
+        self.user = user
+        self.api_base = api_base
+        self.headers = default_headers
+
+    def api_request(self, method, path, **kwargs):
+        """Wrapper users the set user to this API instance's user,
+        provides required headers, appends api base, and passes all else
+        """
+        cluster_request = getattr(self.cluster, method)
+        headers = self.headers
+        if self.user:
+            headers.update(self.user.auth_header)
+        headers.update(kwargs.pop('headers', {}))
+        path = self.api_base + path
+        return cluster_request(path, user=None, headers=headers, **kwargs)
+
+    get = functools.partialmethod(api_request, 'get')
```

Comment:
nit: Would be nice if we didn't have to duplicate these between here and the `Cluster` class